packages/epics/src/treasury/components/assets/deposit-funds.tsx

```diff
@@ -0,0 +1,87 @@
+import Link from 'next/link';
+import { Button, Separator } from '@hypha-platform/ui';
+import { RxCross1 } from 'react-icons/rx';
+import { Text } from '@radix-ui/themes';
+import QRCode from 'react-qr-code';
+import { CopyIcon, CheckIcon, Cross2Icon } from '@radix-ui/react-icons';
+import { copyToClipboard } from '@hypha-platform/ui-utils';
+import { MarkdownSuspense } from '@hypha-platform/ui/server';
+
+interface DepositFundsProps {
+  closeUrl: string;
+  address: string;
+}
+
+const description = `
+Only Base mainnet tokens can be deposited to this address. If you send funds from another network, your tokens may be permanently lost and cannot be recovered.
+
+All deposited tokens are held in the Space's treasury. Any future withdrawals must be approved by passing a proposal in your Space. This ensures transparency and collective decision-making over treasury funds.
+
+Please double-check the following:
+
+- You are depositing Base mainnet tokens, not from any other network.
+
+- You understand that tokens can only be withdrawn after a successful proposal.
+
+- By proceeding, you acknowledge these conditions and confirm your deposit transaction.`;
```

Comment:
These should be the list items @massimobergero has defined, imo

Reply:
> These should be the list items @massimobergero has defined, imo

<img width="446" alt="image" src="https://github.com/user-attachments/assets/001a2a5d-60c0-45e2-98bf-effd7693f0b3" />

but the list items are here, what should the final description be? 🙏